platform/i18n/src/locales/en-US/CineDialog.json

```diff
@@ -1,8 +1,8 @@
 {
-  "Next image": "$t(Common:Play) $t(Common:Image)",
+  "Next image": "$t(Common:Next) $t(Common:Image)",
   "Play / Stop": "$t(Common:Play) / $t(Common:Stop)",
   "Previous image": "$t(Common:Previous) $t(Common:Image)",
   "Skip to first image": "Skip to first $t(Common:Image)",
-  "Skip to last image": "Skip, to last $t(Common:Image)",
```

Comment:
Do not change this here, we need to change it in Locize. You can remove this change from the PR: https://github.com/OHIF/Viewers/blob/48ea7a28e50b845c4939a8aaf2d95eba49d2c102/platform/i18n/src/index.js#L11